SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          | Policy |
+| ------- | ------------------ | ------- |
+| 28.x   | ✅ | Supported branch (6 months from initial release) |
+| 27.x   | ⚠️                | Maintenance branch (6 additional months) |
+| 26.x   | :x:               | Unsupported (volunteers release as needed) |
+
+## Reporting a Vulnerability
+
+As a Java library we often find vulernabilities are reported by downstream (and thus user facing) projects.
+
+* We ask that vulernabilities be reported via our bi-weekly video chat meeting (see developer list for invite).
+* GeoTools is an Open Source Geospatial Foundation project, the [geotools project officer](https://www.osgeo.org/about/board/) can be emailed directly if you are unable to attend meeting above.
+* New: We have enabled GitHub Security Advisories allowing secure reporting of vulnerabilities directly to the projec team
```

Comment:
It does not look like security vulnerabilities can be opened to report issues.
```suggestion
* GitHub Security Advisories list reported / patched vulnerabilities
```

Reply:
that's not how it works; a GH project admin can create a security advisory: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/about-repository-security-advisories

It would be good  to enable private reporting: https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability - there should be a button "Private vulnerability reporting" on https://github.com/geotools/geotools/settings/security_analysis

Reply:
How do we set up a private reporting option? Is it just with GitHub issues?